Localization/de-DE/home.yaml

```
@@ -84,18 +85,18 @@ Für technische Hilfe: ",
 
 #Associated text
 #"Folding refers to the way human protein folds in the cells that make up your body. We rely on the proteins to keep us healthy and they assemble themselves by folding. But when they misfold, there can be serious consequences to a person's health.",
-"Falten bezieht sich auf die Weise, wie sich menschliche Proteine in den Zellen, aus denen Ihr Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, und sie fügen sich durch das Falten zusammen. Aber wenn sie sich falsch falten kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
+"Falten bezeichnet den Vorgang, in dem sich menschliche Proteine in den Zellen, aus denen der menschliche Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, welche sich durch das Falten zusammenfügen. Wenn sich Proteine jedoch falsch falten, kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
 
 #Associated square button
 #"LEARN MORE",
-"ERFAHRE MEHR",
+"MEHR ERFAHREN",
 
 #TEAM bold text
 #TEAM,
 "TEAM",
 
 #Associated text
 #"Folding@home is now based at the Washington University in St. Louis School of Medicine, under the directorship of  Dr. Greg Bowman. Drs. John Chodera (MSKCC) and Vince Voelz (Temple University) are also active in helping manage the project. Together, their three labs are the primary drivers of Folding@home."
-"Folding@home ist derzeit bei der Washington University in St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Zusammen sind ihre drei Labore die hauptsächlichen Treiber von Folding@home."
+"Folding@home ist derzeit bei der Washington University in der St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Dres. John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Ihre drei Labore sind die Hauptantriebskraft von Folding@home."
```

Drs. is short for "doctors" (plural)
```suggestion
"Folding@home ist derzeit bei der Washington University in der St. Louis School of Medicine zu Hause und steht unter der Leitung von Dr. Greg Bowman. Die Doktoren John Chodera (MSKCC) und Vince Voelz (Temple University) unterstützen das Projektmanagement ebenfalls aktiv. Ihre drei Labore sind die Hauptantriebskraft von Folding@home."
```